packages/inventory/src/Stocktake/DetailView/DetailView.tsx

```diff
@@ -40,5 +52,18 @@ export const DetailView: FC = () => {
         />
       )}
     </TableProvider>
+  ) : (
+    <AlertModal
```

Comment:
This is looking so good!

You likely already thought about it but I suppose in the future it would might nice for some sort of alerting 'service'.. I suppose similar to what you have done with the `useNotifications`..  some API like 

```
const { error, notification, warning } = useAlert(..)
```

Where we could just call the callbacks to open this type of modal. 

But I can see that being a decently big job!